### DISCUSSION TITLE
number display

### DISCUSSION THREAD

**Discussion by paarthsiloiya** [SebLague/Digital-Logic-Sim#142] [Q&A] [answered]
In the new version 1.0.2 i cant seem be able to press + or the scroll the scroll wheel to add multiple buttons together that used to disply number in binary. Is their a different button to press now?
<img width="102" alt="image" src="https://user-images.githubusercontent.com/85102021/216802207-cf416300-1bd3-4576-b829-506df632782b.png">
thank you

**Comment by Highlandword9** [comment, marked as answer]
no new button, just disabled during the rework right now. Sebastian said he's working on it.  :]

**Reply by paarthsiloiya** [reply]
ok thanks for the update :)
